tools/king/king.xml

```diff
@@ -0,0 +1,160 @@
+<tool id="king" name="KING" version="2.2.4">
+    <requirements>
+	<requirement type="package" version="2.2.4">king</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    king -b '$genotype' --fam '$family' --bim '$map' $related $duplicate $kinship $ibdseg $ibs $homog
```

Comment:
the issue with why the first test fails is that the `$genotype` file needs to end with the right extension.

At the moment, Galaxy expands `$genotype` to something like `/run/var/path/dataset.dat` which has a `.dat` extension, and so KING thinks it's not of the correct format.

Therefore to solve this for each of our input files, we can do:

    ln -s '$genotype' input.bed &&
    ln -s '$family' input.fam &&
    ln -s '$map' input.bim &&

    king -b input.bed  --fam input.fam --bim input.bim  $otheroptions

This would create soft links (via `ln`)  with the correct file endings


king -b '$genotype' --fam '$family' --bim '$map' $related $duplicate $kinship $ibdseg $ibs $homog